### DISCUSSION TITLE
How to judge whether the dragged file  is a folder or other file

### DISCUSSION THREAD

**Discussion by billyang1222** [react-dropzone/react-dropzone#1231] [Q&A]
How could I get the path of the dragged folder? I want to do nothing if it was a folder

**Comment by rolandjitsu** [comment]
@FireMapleLeaves when you drag'n'drop/select a directory, you will get a `path` property on each item/file, that path will include the directory as well. If the directory is empty, you will get an empty list of files.

I'm not sure if that helps, but there's no other way to tell.
